beard/clustering/wrappers.py

```diff
@@ -36,8 +36,8 @@ class ScipyHierarchicalClustering(BaseEstimator, ClusterMixin):
 
     def __init__(self, method="single", affinity="euclidean",
                  threshold=None, n_clusters=None, criterion="distance",
-                 depth=2, R=None, monocrit=None, scoring=None,
-                 scoring_data=None):
+                 depth=2, R=None, monocrit=None, unsupervised_scoring=None,
+                 supervised_scoring=None, scoring_data=None):
```

Comment:
well, most of these parameters are inherited from the Scipy hierarchal clustering. The difference between the current status and this PR is one additional parameter that is necessary. This only 1 additional paramter will not only fix the current usage issue, but also will allow a hybrid usage of both supervised, when y is available and unsupervised when y is not available.